ludwig/utils/image_utils.py

```diff
@@ -18,8 +18,11 @@
 import os
 import sys
 from math import ceil, floor
+from typing import Tuple, Union
 
 import numpy as np
+# TODO(shreya): Import guard?
```

Comment:
This should be okay, since torch is going to be a required dependency going forward.

Reply:
Sounds good!